aidants_connect_web/tests/test_utilities.py

```diff
@@ -18,3 +29,54 @@ def test_generate_sha256_hash(self):
         )
         self.assertEqual(generate_sha256_hash("123salt".encode()), hash_123salt)
         self.assertEqual(len(generate_sha256_hash("123salt".encode())), 64)
+
+
+@tag("utilities")
+class CreateFirstUserTests(TestCase):
+    def test_dont_create_if_user_exists(self):
+        AidantFactory()
+        self.assertEqual(1, len(Aidant.objects.all()))
+        self.assertIsNone(create_first_user_organisation_and_token())
+        self.assertEqual(1, len(Aidant.objects.all()))
+
+    def test_dont_create_if_organisation_exists(self):
+        OrganisationFactory()
+        self.assertEqual(1, len(Organisation.objects.all()))
+        self.assertIsNone(create_first_user_organisation_and_token())
+        self.assertEqual(1, len(Organisation.objects.all()))
+
+    def test_dont_create_without_all_venv(self):
+        self.assertEqual(0, len(Aidant.objects.all()))
+        self.assertEqual(0, len(Organisation.objects.all()))
+        self.assertIsNone(create_first_user_organisation_and_token())
+        self.assertEqual(0, len(Organisation.objects.all()))
+        self.assertEqual(0, len(Aidant.objects.all()))
+
+    @mock.patch.dict(os.environ, {"INIT_ORGA_NAME": "Donjons et Siphons"})
+    @mock.patch.dict(os.environ, {"INIT_ADMIN_USERNAME": "mario.brossse@world.fr"})
+    @mock.patch.dict(os.environ, {"INIT_ADMIN_PASSWORD": "PEACHforEVER"})
+    def test_dont_create_without_one_venv(self):
+        self.assertEqual(0, len(Aidant.objects.all()))
+        self.assertEqual(0, len(Organisation.objects.all()))
+        self.assertIsNone(create_first_user_organisation_and_token())
+        self.assertEqual(0, len(Organisation.objects.all()))
+        self.assertEqual(0, len(Aidant.objects.all()))
+
+    @mock.patch.dict(os.environ, {"INIT_ORGA_NAME": "Donjons et Siphons"})
+    @mock.patch.dict(os.environ, {"INIT_ADMIN_USERNAME": "mario.brossse@world.fr"})
+    @mock.patch.dict(os.environ, {"INIT_ADMIN_PASSWORD": "PEACHforEVER"})
+    @mock.patch.dict(os.environ, {"INIT_TOKEN": "12345"})
```

Comment:
```suggestion
    @mock.patch.dict(os.environ, {"INIT_TOKEN": "123456"})
```


Six chiffres le token, sinon tu vas avoir des problèmes